wa_lap_identification.sql

```diff
@@ -0,0 +1,240 @@
+/***********************************************************************************************************************
```

Comment:
Good documentation!

Reply:
ha! You wrote pretty much all of it. I just substituted LAP in.